.github/workflows/backend.yml

```diff
@@ -61,3 +61,5 @@ jobs:
                  -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
       - name: Run Unit tests
         run: ./mvnw clean verify -B -Dmaven.test.skip=false -Dgpg.skip=true
+      - name: Check dependency license
+        run: sudo tools/dependencies/checkLicense.sh
```

Comment:
If the `mvnw` command can run without sudo permission, the `checkLincense.sh` command should not.

Reply:
hi, this is because there is no permission, but I see that other projects seem to be able to do so. I am still looking for a problem, thank you for your comment. Currently this PR is still in WIP status, you can review it after I finish it.